bedrock/security/views.py

```diff
@@ -175,8 +175,7 @@ def get_redirect_args(self, url_component):
         match = self.prod_ver_re.match(url_component)
         if match:
             product, version = match.groups()
-            version = "{0}.{1}".format(*version)
-            return dict(viewname="security.product-version-advisories", kwargs={"product": product, "version": version})
+            return dict(viewname="security.product-version-advisories", kwargs={"product": product, "version": ".".join(version)})
```

Comment:
Idle/no-op thought: looks like the old formatting would explode if `version` was a iterable with more than two elements, whereas the new version will handle any length of iterable.

Reply:
Explode meaning expand? Or explode meaning throw an error? I tried to simplify this one when I stumbled on it.

Reply:
I may update this since these are different:
```
>>> version = ["42", "4", "2"]
>>> ".".join(version)
'42.4.2'
>>> "{0}.{1}".format(*version)
'42.4'
```

Reply:
Gonna run with:
`f"{version[0]}.{version[1]}"`